src/js/main.js

```diff
@@ -44,7 +44,7 @@ export function init(el, context, config) {
 
         const chapterButtons = builder.querySelector('.docs--chapters');
         chapters.forEach( function(chapter){
-          chapterButtons.innerHTML += '<li data-sheet-timestamp="'+ chapter.chapterTimestamp +'">' + chapter.chapterTitle + '</li>';
+          chapterButtons.innerHTML += `<li data-sheet-timestamp='${chapter.chapterTimestamp}' data-link-name='${config.sheetChapter} | ${chapter.chapterTitle}'>${chapter.chapterTitle}</li>`;
```

Comment:
I believe they are just strings and you do whatever post processing on them later. It might be a bit easier to parse if non spaced though, so I can do that transform on them.